.spelling

```diff
@@ -72,6 +72,7 @@ Exfiltrating
 ExternalName
 Firebase
 FitStation
+FIXED_WINDOW
```

Comment:
Shouldn't be in the dictionary, put backticks around use instead.

Reply:
Fixed